compiler/sigmatch.nim

```diff
@@ -993,6 +993,11 @@ proc typeRel(c: var TCandidate, f, aOrig: PType,
   result = isNone
   assert(f != nil)
 
+  when declared(deallocatedRefId):
```

Comment:
that's new from this PR, should be `nimDeallocatedRefId`

Reply:
It's ``declared``, not ``defined``, but ok.

Reply:
oh ya, my bad